lib/data_hygiene/attachment_attribute_not_fixable.rb

```diff
@@ -0,0 +1,3 @@
+module DataHygiene
```

Comment:
Minor: could we make this a subclass of `AttachmentAttributeUpdater`? This keeps the number of classes/files down, and makes it clear it's only used in this context.